client/server.js

```diff
@@ -35,7 +34,6 @@ server.app.get('/comments.json', function(req, res) {
 server.app.post('/comments.json', function(req, res) {
   console.log('Processing comment: %j', req.body.comment);
   console.log('(shhhh...napping 1 seconds)');
-  sleep.sleep(1);
```

Comment:
@justin808, I removed because we don't need this if we the minimum delay func.
Let's talk about this below.